query/src/servers/http/clickhouse_handler.rs

```diff
@@ -139,18 +139,45 @@ async fn execute(
             rb
         }
     };
+
+    let mut blocks: Vec<Result<Vec<u8>>> = vec![];
+    for _ in 0..2 {
```

Comment:
How about leaving some comments to explain why `0..2` here?

Reply:
OK